src/sage_setup/autogen/interpreters/__main__.py

```diff
@@ -1,9 +1,18 @@
-from __future__ import print_function, absolute_import
+from __future__ import absolute_import, print_function
```

Comment:
this line can be deleted, it's no longer needed with python 3.

Reply:
Done